NorthstarDLL/plugins/plugins.cpp

```diff
@@ -188,8 +188,31 @@ std::optional<Plugin> PluginManager::LoadPlugin(fs::path path, PluginInitFuncs*
 	return plugin;
 }
 
+inline void findPlugins(fs::path pluginPath, std::vector<fs::path>& paths)
+{
+	if (fs::exists(pluginPath) && fs::is_directory(pluginPath))
```

Comment:
( just a nitpick ) you could inverse this if statement and return to reduce nesting.